node/src/types/chainspec/network_config.rs

```diff
@@ -25,23 +21,6 @@ pub struct NetworkConfig {
     pub(crate) accounts_config: AccountsConfig,
 }
 
-impl NetworkConfig {
-    /// Returns a vector of chainspec validators' public key and their stake.
-    pub fn chainspec_validator_stakes(&self) -> Vec<(PublicKey, Motes)> {
```

Comment:
This will actually be used by fast sync again, so we might as well keep it to avoid more merge conflicts.

Reply:
I would say that it's not the reason to keep an unused code around but I know this will be the cause of a lot of tension so I will bring it back.

Reply:
Should be brought back in [`4307d72` (#1757)](https://github.com/casper-network/casper-node/pull/1757/commits/4307d726c27c8a225fe43d21d577450912a9e229)